components/suggestion/command_provider/command_provider.tsx

```diff
@@ -125,7 +154,11 @@ export default class CommandProvider extends Provider {
         const command = pretext.toLowerCase();
         Client4.getCommandsList(getCurrentTeamId(this.store.getState())).then(
             (data) => {
-                let matches: CommandSuggestionItem[] = [];
+                let matches: AutocompleteSuggestion[] = [];
+                if (appsEnabled(this.store.getState())) {
+                    const appCommandSuggestions = this.appCommandParser.getSuggestionsBase(pretext);
```

Comment:
When would we run into a case where pretext isn't an app command but we still have suggestions for it? It seems weird that we have a code path specific for suggestions coming from the AppCommandParser and then this one that merges results from regular commands and app commands here

Reply:
Oh, I think I see. The base commands are just the first level of them (like "/agenda" in my previous example), and `isAppCommand` only returns true once the user has typed one of those base commands. Is that correct?

Reply:
Yes your last comment is correct. When `isAppCommand` returns true, that tells the command provider "let me handle all the suggestions here." Maybe something to include in a comment to be more clear.